doc/source/whatsnew/v1.0.0.rst

```diff
@@ -891,7 +891,7 @@ Other
 - Fixed :class:`IntegerArray` returning ``inf`` rather than ``NaN`` for operations dividing by 0 (:issue:`27398`)
 - Fixed ``pow`` operations for :class:`IntegerArray` when the other value is ``0`` or ``1`` (:issue:`29997`)
 - Bug in :meth:`Series.count` raises if use_inf_as_na is enabled (:issue:`29478`)
-
+- Add tests for keyword argument ``if_exists`` in :meth:`DataFrame.to_gbq` (:issue:`29598`)
```

Comment:
No note needed as this isn't a user-facing change

Reply:
ping 
@WillAyd